src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -26,7 +26,7 @@ public void ConfigureServices(IServiceCollection services)
         {
 #if Auth
             #warning Please provide a valid secret provider, for example Azure Key Vault: https://security.arcus-azure.net/features/secrets/consume-from-key-vault
```

Comment:
Can't we just use `#error` instead?

Reply:
Ah, yes; but we still have to remove the contants to make it buildable for dev. Hang on.

Reply:
That's fine, but I'm more of a fan of the `null` approach as it's up to them to add what they want.

DEV is our issue :D